README.md

```diff
@@ -11,10 +11,10 @@ urlFragment: get-set-keyvault-secrets-managed-id-python
 
 # How to set and get secrets from Azure Key Vault with Azure Managed Identities and Python
 
-## SDK Versions
-In this sample, you will find the following folders:
-* **v3** - references Key Vault SDK v3
-* **v4** - references Key Vault SDK v4
+## This sample shows how to do the following operations of Key Vault secret with Key Vault SDK
+- Get Key Vault MSIAuthentication or ServicePrincipalCredentials
+- Create a Key Vault client
+- Get an existing secret
 
 ## Background
 For service to service authentication, the approach involved creating an Azure AD application and associated credential, and using that credential to get a token. While this approach works well, there are two shortcomings:
```

Comment:
Regarding line 41: there are other required fields when deploying the template, and we might want to mention them as well (resource group, website name, key vault name). This might suggest to a user that only the secret needs to be specified.